.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
```

Comment:
Should we support all Ruby versions that are not EOL? It looks like that's the default policy in `dry-rb` repos. See https://github.com/dry-rb/dry-monads/blob/a3895a55a449e415009d9f0aae25d5b80c86fd16/.github/workflows/ci.yml#L28.

Reply:
Update: ah! I just looked at the Rakefile again, so maybe we want just `rake spec` for the matrix testing step, and then a single step that runs `rake code_quality` on just the latest Ruby.

Reply:
Sounds good, happy to see this done as a follow up!